### PR TITLE
Part 2

### PR DESCRIPTION
**Pull request by Kingjosh007**
In this version, I improved the mobile version of my portfolio:

- I added a Recent Works section, with a few recent projects I worked on.
- I provided links to my Github, LinkedIn, AngelList, Twitter, and Medium profiles
- I updated the README file for a better description of my project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,74 +11,209 @@
...
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">brain.js</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">redis</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">Adaboost</div>
+                    </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap this with `ul` and each of them `li`
 The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 2]
```diff
@@ -11,74 +11,209 @@
...
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">electron.js</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">indexedDB</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">html</div>
+                    </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap this with `ul` and each of them `li`
 The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 3]
```diff
@@ -11,74 +11,209 @@
...
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">html</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">bootstrap</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">Ruby</div>
+                    </div>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap this with `ul` and each of them `li`
 The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 4]
```diff
@@ -11,74 +11,209 @@
...
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us wrap this with `ul` and each of them `li`
 The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li

File: index.html [hunk 5]
```diff
@@ -11,74 +11,209 @@
...
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">indexedDB</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">html</div>
+                    </div>
+                    <button class="projectDetailsBtn">See Project</button>
+                </div>
+
```
Comment (Emmyn5600) [inline index.html:162]: The button (e.g., "See project") is a link a or button tag nice, and follow the button styling: Active, Hover and Pressed, let us add `cursor:pointer` here and add `hover color`

File: index.html [hunk 6]
```diff
@@ -11,74 +11,209 @@
...
+                        <div class="technology">html</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">bootstrap</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">Ruby</div>
+                    </div>
+                    <button class="projectDetailsBtn">See Project</button>
+                </div>
```
Comment (Emmyn5600) [inline index.html:187]: The button (e.g., "See project") is a link a or button tag nice, and follow the button styling: Active, Hover and Pressed, let us add `cursor:pointer` here and add `hover color`

File: index.html [hunk 7]
```diff
@@ -11,74 +11,209 @@
...
```
Comment (Emmyn5600) [inline index.html:213]: The button (e.g., "See project") is a link a or button tag nice, and follow the button styling: Active, Hover and Pressed, let us add `cursor:pointer` here and add `hover color`

File: index.html [hunk 8]
```diff
@@ -11,74 +11,209 @@
...
+                        <div class="technology">brain.js</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">redis</div>
+                        <div class="separator"><img src="assets/svg/vertical_line.svg"
+                                alt="Line separing the technologies used in a project"></div>
+                        <div class="technology">Adaboost</div>
+                    </div>
+                    <button class="projectDetailsBtn">See Project</button>
+                </div>
```
Comment (Emmyn5600) [inline index.html:136]: The button (e.g., "See project") is a link a or button tag nice, and follow the button styling: Active, Hover and Pressed, let us add `cursor:pointer` here and add `hover color`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Kingjosh007 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

- Please Add the descriptive PR before you request for a re-review
- Use the descriptive feature branch for example `work_section` not `part-2`  

 * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 8 above)
Comment (Kingjosh007) [reply index.html:136]: OK, doing it right away.

**Comment by Kingjosh007** [comment]
I just tried renaming the branch as you suggested, but all the PRs were automatically closed, including this one. I just restored the branch then. Sorry for keeping the same name for this branch. Otherwise, this PR will vanish.

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @Kingjosh007 

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer 
Well done

- To highlight:
   * #### You have the descriptive Readme:heavy_check_mark:
   * #### Your commits look more professional:heavy_check_mark:
   * #### Linters are passing successfully.:heavy_check_mark:
   * #### You have descriptive PR:heavy_check_mark:
   * #### Your project looks awesome:heavy_check_mark:
   
#### Optional suggestion
   - Use a descriptive feature branch for example `work-section` not `part-2`
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**